Package.swift

```diff
@@ -374,5 +375,23 @@ let package = Package(
         .unsafeFlags(["-I\(javaIncludePath)", "-I\(javaPlatformIncludePath)"])
       ]
     ),
+
+    .executableTarget(
```

Comment:
This won't work, because we'll get the following on macOS:

> error: the executable 'JavaApiCallBenchmarks' requires macos 10.15, but depends on the product 'Benchmark' which requires macos 13.0; consider changing the executable 'JavaApiCallBenchmarks' to require macos 13.0 or later, or the product 'Benchmark' to require macos 10.15 or earlier.

The way to do this is to make a directory `Benchmarks/` and make a new `Package.swift` in there, and refer it back into `..` for the library dependency

Reply:
Here's an example how to set it up: https://github.com/apple/swift-nio/blob/main/Benchmarks/Package.swift

Reply:
I think I've set it up correctly now -- thanks for the example file! It helped a bunch.